lib/mongoose_push_web/router.ex

```diff
@@ -8,14 +8,14 @@ defmodule MongoosePushWeb.Router do
 
   pipeline :swagger_json do
     plug(:accepts, ["json"])
-    plug(OpenApiSpex.Plug.PutApiSpec, module: MongoosePushWeb.ApiSpec)
+    plug(MongoosePushWeb.SwaggerPlug, module: MongoosePushWeb.ApiSpec)
```

Comment:
```suggestion
    plug(MongoosePushWeb.Plug.MaybePutApiSpec, module: MongoosePushWeb.ApiSpec)
```

This is not technically swagger, this is OpenAPI. And the original naming is actually quite nice, as this plug "puts" the spec into the endpoint. There is also plug that validates agains it, so `SwaggerPlug` is simply too generic in my opinion.